lib/membrane/rtp/outbound_tracking_serializer.ex

```diff
@@ -173,7 +181,7 @@ defmodule Membrane.RTP.OutboundTrackingSerializer do
 
     payload =
       RTP.Packet.serialize(%RTP.Packet{header: header, payload: buffer.payload},
-        align_to: state.alignment
+        align_to: if(is_padding_packet?, do: @padding_packet_size, else: state.alignment)
```

Comment:
I would leave this as it was and handle padding packets inside Packet.serialize. This way we will be able to create padding packets that are 255 + 1 + header bytes long in total and we can focus on testing Packet.serialize instead of OutboundSerializingTracker